python/cuml/cuml/dask/ensemble/__init__.py

```diff
@@ -13,15 +13,5 @@
 # limitations under the License.
 #
 
-from cuml.internals.import_utils import has_dask
-import warnings
-
-if has_dask():
-    from cuml.dask.ensemble.randomforestclassifier import (
-        RandomForestClassifier,
-    )
-    from cuml.dask.ensemble.randomforestregressor import RandomForestRegressor
-else:
-    warnings.warn(
-        "Dask not found. All Dask-based multi-GPU operation is disabled."
-    )
```

Comment:
This previous warnings code would never run. If a dep wasn't present, the error would occur far before here. Better to just use normal imports.